### ISSUE TITLE
Need help, unable to update test cases

### ISSUE THREAD

**Issue by needhamm** [gianlucamangiapelo/wdio-azure-devops-service#4]
Hello, I am hoping that you could give me a hand in getting this service working. I have added the service as per https://webdriver.io/docs/gmangiapelo-wdio-azure-devops-service/ and as far as I can tell have it setup correctly but test cases are not being updated in Azure.

I have looked through issue #1 but have still not been able to get it working. Below is the setup that I have in wdio and Azure. Any help would be greately appreciated, thanks in advance.

PAT setup with the following:
![image](https://user-images.githubusercontent.com/110604538/182830155-b6f2bd7c-db88-423e-a5cd-0c2be6cf6e6e.png)

wdio.conf.js setup as so:

```
    services: [
        [driverService], 
        [AzureDevopsService,
            {
            pat: 'pat',
            organizationUrl: 'https://dev.azure.com/org',
            projectId: 'projectid',
            planId: 30584,
            suiteId: 37063,
            caseIdRegex: '@?[cC](\d+)',
            runName: 'AzureDevops Test',
            },
        ],
    ],
```

When querieng the through Postman I am able to return test cases with no issues using the pat, org, projectId, planId and suiteId
GET https://dev.azure.com/{org}/{projectid}/_apis/test/Plans/30584/suites/37063/testcases?api-version=5.0

```
"testCase": {
                "id": "37064",
                "url": "https://dev.azure.com/{org}/_apis/wit/workItems/37064",
                "webUrl": "https://dev.azure.com/{org}/web/wi.aspx?pcguid=########-####-####-####-############&id=37064"
            },

```

Test case setup, I have tried with lower and upper case C:
```
describe('Test 1', () => {


    it( 'C37064 Test case', async () => {
        
  
    });

});
```

After running the test Azure is not updated:
![image](https://user-images.githubusercontent.com/110604538/182833701-0d0d0ad1-0415-45af-80c3-a24d5919fdf6.png)

I have set the logging to:
```
logLevel: "trace",
outputDir: path.join(__dirname, "/logs"),
```

But can see no mention of this service in any of the logs.

**Comment by gianlucamangiapelo** [comment]
Hi @needhamm, 
seems that your configuration is okay.

You can try to remove from wdio.conf.js the `caseIdRegex` property, because you can use the default regex.

Here the configuration that you can try:
```js
services: [
        [driverService], 
        [AzureDevopsService,
            {
            pat: 'pat',
            organizationUrl: 'https://dev.azure.com/org',
            projectId: 'projectid',
            planId: 30584,
            suiteId: 37063,
            runName: 'AzureDevops Test',
            },
        ],
    ],
```
Let me know if you have some progress on that.

Thanks

**Comment by needhamm** [comment]
Hi @gianlucamangiapelo 

Thanks for getting in touch. I have tried removing the regex but no change. 

There is still no mention of the service in the wdio.log when logging set to trace. Would this suggest that the service isnt being called for some reason?

**Comment by gianlucamangiapelo** [comment]
@needhamm Probably is not loaded.

At the top of the `wdio.conf.js` are you importing 
`import AzureDevopsService from "@gmangiapelo/wdio-azure-devops-service";`

 the plugin is listed in your dependencies in package.json?

**Comment by daveplummer20** [comment]
Hi.
I'm working with @needhamm on this and have now managed to get the service to update tests in the suite to In Progress, however no test result is being added to the test case.
![image](https://user-images.githubusercontent.com/36714967/188666169-805bdb72-4034-4184-a7c0-29debd69684d.png)

I can see the service being initialised at the beginning of the log, but can't see any reference to it after that:
@wdio/utils:initialiseServices: initialise custom service "AzureDevopsService"

Is there anything else we can check to try and debug?

Thanks

**Comment by daveplummer20** [comment]
I managed to get this working.
We're using jasmine which doesn't seem to include the title on the test object, which from what I could gather from your code is one of the things that is used to parse the test case id.
For now I've added the following to the afterTest hook and the test cases are now being updated with the correct result.
test.title = test.description;

Might be something to think about adding in a future version?
